t/14-isotovideo.t

```diff
@@ -14,9 +14,12 @@ my $pool_dir     = "$toplevel_dir/t/pool";
 
 sub isotovideo {
     my (%args) = @_;
-    $args{opts}      //= '';
-    $args{exit_code} //= 1;
-    system("perl $toplevel_dir/isotovideo -d $args{opts} 2>&1 | tee autoinst-log.txt");
+    $args{default_opts} //= 'backend=null';
+    $args{opts}         //= '';
+    $args{exit_code}    //= 1;
+    my $cmd = "perl $toplevel_dir/isotovideo -d $args{default_opts} $args{opts} 2>&1 | tee autoinst-log.txt";
+    note("Starting isotovideo with: $cmd");
+    system($cmd);
```

Comment:
Maybe merge `note`/`system` here a la `is(system(...`?

Reply:
I don't understand what you mean. "note" is there to just print some debug message on `prove -v`, `system` is executing something. There is nothing to compare `is` against here.

Reply:
The next line already does it, and checks that the exit code is `0`. That´s why I thought of it.